cmd/blockchaincmd/deploy.go

```diff
@@ -643,6 +657,109 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 				return err
 			}
 		}
+
+		bar := progressbar.NewOptions(1002,
+			progressbar.OptionSetWriter(ansi.NewAnsiStdout()),
+			progressbar.OptionEnableColorCodes(true),
+			progressbar.OptionSetElapsedTime(false),
+			progressbar.OptionSetWidth(15),
+			progressbar.OptionSetDescription("[cyan][1/3][reset] Waiting for validators info update on P-Chain..."),
+			progressbar.OptionSetTheme(progressbar.Theme{
+				Saucer:        "[green]=[reset]",
+				SaucerHead:    "[green]>[reset]",
+				SaucerPadding: " ",
+				BarStart:      "[",
+				BarEnd:        "]",
+			}))
+		for i := 0; i < 1000; i++ {
```

Comment:
nit:we should abstract this away. ux func to accept direction and we split into steps and make bar progress

Reply:
done